### DISCUSSION TITLE
Need help to generate code for a module without table in migration.cs file

### DISCUSSION THREAD

**Discussion by moseur** [serenity-is/Serenity#6538] [Help]
Dear Specialist,

I am newbie in serenity family. Last two month I am enjoying serenity platform. Everything is going fine. But Now I am stack in a problem. I have an inventory Module also this module have a class stock ( where i will show product stock status ). This stock process only have calculation. I dont need db.table for this class. How Can I genereate code for stock class identifier. I have tried, But I am stack.

Thanks
Moseur

**Comment by dfaruque** [comment]
I think there is no such thing in serenity. to generate code you need a table or view.

however, you may copy pasting from your existing code and rename file name then find and replace the identifier using visual studio replace tool

**Reply by moseur** [reply]
Thanks Faruque Vai,

For your quick response. I have tried your phone number. I found your number switched off. By the way, I will be happy to talk to you. 

Thanks again.  I will follow your instruction.

**Reply by moseur** [reply]
Dear Faruque Vai,

Can you help me , How can I create a view with code generating. Without table is it possible to create a view?

Thanks
Moseur

**Reply by dfaruque** [reply]
I recommend to use pure asp.net MVC for this kind of page. you do not need to generate it. you need to write it yourself.

**Reply by dfaruque** [reply]
take dashboard page as a sample

**Reply by moseur** [reply]
Thanks Again. Let me try.

**Reply by edwardch** [reply]
I have created temporary views and then used them to do the code gen, as that was (for my case) easier than writing my own classes from scratch. 

Once you've generated the code you can drop your view, and then just tweak your table name and handlers, etc...
